openbb_terminal/stocks/options/yfinance_model.py

```diff
@@ -128,27 +128,29 @@ def get_option_chain_expiry(
     # Add in greeks to each df
     # Time to expiration:
     dt = (
-        datetime.strptime(expiry, "%Y-%m-%d") + timedelta(hours=16) - datetime.now()
-    ).total_seconds() / (60 * 60 * 24)
+        (datetime.strptime(expiry, "%Y-%m-%d") - datetime.now()).total_seconds()
+        + 16 * 60 * 60
+    ) / (60 * 60 * 24)
```

Comment:
Just curious why you decided to change from the time delta (which I added in #3792).  Technically not wrong, but using time delta is more readable imo.

Reply:
Oh, I worked on an old version and then just overwrote the changes when I fetched. I'll switch back 👍